test/admin-hpa/Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'rb-readline'
 gem 'pry'
 gem 'require_all'
 gem 'rspec'
-gem 'selenium-webdriver'
+gem 'selenium-webdriver', '4.0.0.alpha1'
```

Comment:
Alpha?

Reply:
@jon-shipley -  its the only version that works with it atm.

Reply:
Okay.  Let's keep an eye on it then and upgrade when it's stable 😓

Reply:
@jon-shipley - For sure, using this version at least gets the tests up and running. Agree to keep an eye on it